model/src/main/java/com/expediagroup/streamplatform/streamregistry/model/ClusterValue.java

```diff
@@ -1,4 +1,5 @@
-/* Copyright (C) 2018-2019 Expedia, Inc.
+/**
```

Comment:
This "move" is all foobar.  Can you spend a little time investigating if it makes more sense to maintain history as @massdosage suggests?

Otherwise, I'd be more of the opinion of breaking this up into two PRs.
One PR - DELETE
One PR - ADD

To make it more clean.

Reply:
ok I'll see if I can fix the dodgy diffs

Reply:
I wouldn't spend much time on it. If its too wonky, go with the more draconian delete, followed up with an add.

Reply:
I've rebased to 3 staged commits and force pushed. I managed to easily resolve the cases where it thought it was a rename between 2 unrelated files. The real renames I haven't been able to resolve. I've tried several solutions reported on various forums. They remain as delete/add. A lot of these are going to be deleted in subsequent PRs anyway. I'm happy with the current state.

Reply:
Actually, even though `KStreamsProcessorListener.java` was deleted in commit 1 and `StreamRegistryApp.java` was added in commit 2, it's still showing up in the PR as a rename. I give up.

Reply:
This is fine.
👍